core/src/main/java/com/yahoo/ycsb/workloads/RestWorkload.java

```diff
@@ -0,0 +1,300 @@
+/**

```

Comment:
I think more time needs to be spent removing this RestWorkload file. I think it overlaps with CoreWorkload.

The format of CoreWorkload and RestWorkload of how it documents the properties and defines variables is different I think. Maybe try to make them more similar?